src/components/forms/login-form.js

```diff
@@ -79,11 +79,11 @@ function LoginForm({ onSubmit, withTitle }) {
             ))}
 
             <Text textAlign="right" mb={2}>
-              <NextLink href="/account/forgot-password">
-                <Link textAlign="right" color="linkBlue">
-                  Forgot password?
-                </Link>
-              </NextLink>
+              {/* <RouterLink href="/account/forgot-password"> */}
```

Comment:
why not delete this ?

Reply:
i will reactive them when we go back to work with login-form and other old pages. they are necessary links but I didn't check them in client-sided create-react-app so I don't know how to implement our software yet.